.github/workflows/docker.yaml

```diff
@@ -38,9 +45,13 @@ jobs:
           CRAWLERS_API_KEY=secret
           GITHUB_TOKEN=123
           EOF
+      - name: Build monocle-exe container
+        run: "nix build .#containerMonocle; docker load < ./result"
+      - name: Build docker image
+        run: |
```

Comment:
It seems we issue twice the same command

Reply:
What do you mean? The first docker load setup the "monocle-exe" image, and docker build produces the final "monocle" image, with the web assets.

Reply:
https://github.com/change-metrics/monocle/blob/6099cfa475588a6497f90e3f952ad710949bee6c/.github/workflows/docker.yaml#L50-L55

Reply:
oh ok, then 8f9ca41 fix that :)